src/main/resources/db/migration/V006__Add_status_column.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE letters
+ADD COLUMN status VARCHAR(256);
```

Comment:
fill the values here base on dates set

Reply:
I'd wait with this filling this column until the service populates the status itself. Then, when we apply that script, be can be sure all records have it set properly. Otherwise, the service will create new ones without status.

Reply:
it already does

Reply:
But not the producer. The two services need to have identical sets of migration scripts.

Reply:
ok

Reply:
Can we write an integration test for these migrations? We should be able to write a test that runs the migrations up to this one, populates some data and then runs the last two.